test/unit/data/fake_names_index_template.json

```diff
@@ -0,0 +1,395 @@
+{
+  "order": 0,
+  "template": "names",
+  "settings": {},
+  "mappings": {
+    "_default_": {
+      "dynamic_templates": [
+        {
+          "string_fields": {
```

Comment:
I need the explicit mappings otherwise I'll get mapping exceptions when indexing the data. For instance, some of the ZipCode values are actually strings, but Elasticsearch will map is as a long if the first value it sees looks like a number. So ZipCode needs to be explicitly mapped as a string.

Some of the fields might not strictly require explicit mappings, but to make the tests as robust and repeatable as possible, I went ahead and mapped everything.